input/winscap.c

```diff
@@ -132,6 +304,44 @@ struct {
     {AUDCLNT_E_UNSUPPORTED_FORMAT, L"Requested sound format unsupported"},
 };
 
+void write_silent_frame(struct audio_data *audio, IAudioCaptureClient *pCapture,
+                        UINT32 numFramesAvailable, UINT32 *packetLength) {
+	// Send one silent frame to the spectrometer
+	int silent_channels = audio->channels;
+	int silent_bytes = silent_channels * sizeof(int16_t); // 16-bit PCM
+	int16_t silence[2] = {0};
+
+	write_to_cava_input_buffers(silent_channels, (unsigned char *)silence, audio);
+	pCapture->lpVtbl->ReleaseBuffer(pCapture, numFramesAvailable);
+	pCapture->lpVtbl->GetNextPacketSize(pCapture, &packetLength);
```

Comment:
is it maybe just `packetLength` here, without the `&`?

Reply:
or maybe better remove the * from the write_silent_frame packetLength function argument and remove the & when calling the function.